lib/ansible/module_utils/network/fortios/fortios.py

```diff
@@ -106,11 +106,11 @@ def schema(self, path, name, vdom=None):
 
         if status == 200:
             if vdom == "global":
-                return json.loads(result_data.decode('utf-8'))[0]['results']
+                return json.loads(result_data)[0]['results']
```

Comment:
These should all be changed like so
```suggestion
                return json.loads(to_text(result_data))[0]['results']
```
`to_text` is importable from `ansible.module_utils._text`.

While we're at it your current uses of `to_native` (imported from the same place) should also be changed to `to_text`